cmd/medius/common/registry.go

```diff
@@ -25,43 +25,50 @@ type Entry struct {
 var staticRegistry = []Entry{
 	{
 		Artifacts: []api.Artifact{
-			centos.New("8.4", nil),
+			centos.New("8.4", "x86_64", nil),
+			centos.New("8.4", "aarch64", nil),
```

Comment:
CentOS 8 is already EOL, why are we still shipping these containerdisks?

https://www.centos.org/centos-linux-eol/

Reply:
I'm not aware of it, thanks for taking care of it in https://github.com/kubevirt/containerdisks/pull/92.